webapp/src/pages/Settings.tsx

```diff
@@ -190,6 +190,22 @@ const Settings: React.FC = () => {
     </Box>
   );
 
+  const displayArgumentsList = (name: string, argEntries: any[]) => (
+    <Box display="flex" flexDirection={"column"}>
```

Comment:
You can fix issue 2 of https://github.com/ServiceNow/azimuth/pull/367/files#r1068648552 by changing
```suggestion
    <Box display="grid">
```